scim2/scim2_connector.bal

```diff
@@ -169,8 +167,7 @@ function ScimConnector::getListOfGroups() returns (Group[]|error) {
     var res = httpEP->get(SCIM_GROUP_END_POINT, message = request);
     match res {
         error err => {
-            Error = { message: failedMessage + err.message, cause: err.cause };
-            return Error;
+            return err;
```

Comment:
You can remove the `{}` in the match pattern for error since it only contains single line. 
Example: 
```ballerina
error err => return err;
```